azure-functions/acbs-function/activity-update-facility-loan/index.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Facility loan amendment DAF
+ * ***************************
+ * This DAF (Durable Activity Function) is never invoked directly.
+ * It is invoked via DOF (Durable Orchestrator Function).
+ *
+ * Pre-requisites
+ * --------------
+ * 0. 'npm install durable-functions'
+ * 1. Durable  HTTP starter function (acbs-http)
+ * 2. Durable Orchestrator function (DOF) (acbs-amend-facility)
+ *
+ * ------------------
+ * HTTP -> DOF -> DAF
+ * ------------------
+ */
+
+const moment = require('moment');
+const api = require('../api');
+const { isHttpErrorStatus } = require('../helpers/http');
+const { findMissingMandatory } = require('../helpers/mandatoryFields');
+
+const mandatoryFields = [
+  'expiryDate',
+];
+
+const updateFacilityLoan = async (context) => {
+  try {
+    const {
+      loanId,
+      facilityId,
+      acbsFacilityLoanInput,
+    } = context.bindingData;
+
+    const missingMandatory = findMissingMandatory(acbsFacilityLoanInput, mandatoryFields);
+
+    if (missingMandatory.length) {
+      return Promise.resolve({ missingMandatory });
```

Comment:
can we use `await` instead above?

Reply:
This has always been `return` I have just formatted thus lines are being highlighted.